app/services/change_schedule.rb

```diff
@@ -9,6 +9,7 @@ class ChangeSchedule
   attribute :course_identifier
   attribute :schedule_identifier
   attribute :cohort, :integer
+  attribute :attempt_to_change_cohort_leaving_billable_declarations, :boolean, default: false
```

Comment:
Do we need this security attribute? 
Nothing will change until we set a cohort as payments-frozen in the database, no?

Reply:
We need this because the `ChangeSchedule` service is exposed via the lead provider API and we don't necessarily want them to be able to use it to perform this kind of cohort change yet

Reply:
That's fine but if we don't set 2021 as payments-frozen yet, even if they try that cohort change the code we have written won't  allow it (even if we remove `attempt_to_change_cohort_leaving_billable_declarations`)
In any case, fine by me if you want to keep it for extra security reasons.

Reply:
Yeah I understand that it won't let it happen until the payments frozen is switched on, but we want this as an extra safety net as it looks like we may not want to allow providers to use the service for this purpose (but we want the option to trigger it via other means, so that's where the flag will come in)